biothings/hub/datatransform/__init__.py

```diff
@@ -7,8 +7,8 @@
 # only for testing purpose, this one is not optimized
 #from biothings.hub.datatransform.datatransform_serial import DataTransformSerial
 
-# this involved biothings client dependency, skip it for now
```

Comment:
this comment is legit, I removed those lines to avoid having biothings_clients as a dependency for the hub (you would require it just to import biothings.hub.*, and pip install biothings_clients installs a very specific version of biothings)